2d_solver/msh_parser.py

```diff
@@ -0,0 +1,300 @@
+            (line1_xs, line1_ys) = zip(*line)
+            axes_handler.add_line(lines.Line2D(line1_xs, line1_ys, linewidth=grid_width, color=grid_color))
+    
+    #Plot the horizontal lines
+    for n in np.arange (0, N):
+        for m in np.arange (1, N):
+            line = [array3d[m][n].tolist(), array3d[m-1][n].tolist()]
+            (line1_xs, line1_ys) = zip(*line)
+            axes_handler.add_line(lines.Line2D(line1_xs, line1_ys, linewidth=grid_width, color=grid_color))
+    
+    return
+
+
+def plot_element_boundary(x_nodes, y_nodes, axes_handler,
+                          grid_width = 2., grid_color = 'blue'):
+    '''
+    Plots the boundary of a given :math:`2^{nd}` order element.
+    
+    Parameters
+    ----------
+    
+    x_nodes : np.ndarray [8]
+              :math:`x` nodes of the element.
+             
+    y_nodes : np.ndarray [8]
+              :math:`y` nodes of the element.
+             
+    axes_handler : matplotlib.axes.Axes
+                   The plot handler being used to plot the element grid.
+                   You may generate it by calling the function pyplot.axes()
+                   
+    grid_width   : float
+                   Grid line width.
+                 
+    grid_color   : str
+                   Grid line color.
+
+    Returns
+    -------
+    
+    None
+
+    '''
+    
+    xi  = np.linspace(-1, 1, 20)
+    eta = np.linspace(-1, 1, 20)
+    
+    left_edge   = np.zeros([xi.size, 2])
+    bottom_edge = np.zeros([xi.size, 2])
+    right_edge  = np.zeros([xi.size, 2])
+    top_edge    = np.zeros([xi.size, 2])
+    
+    left_edge[:, 0]   = isoparam.isoparam_x(x_nodes, -1., eta)
+    bottom_edge[:, 0] = isoparam.isoparam_x(x_nodes, xi, -1)
+    right_edge[:, 0]  = isoparam.isoparam_x(x_nodes, 1., eta)
+    top_edge[:, 0]    = isoparam.isoparam_x(x_nodes, xi, 1.)
+    
+    left_edge[:, 1]   = isoparam.isoparam_y(y_nodes, -1., eta)
+    bottom_edge[:, 1] = isoparam.isoparam_y(y_nodes, xi, -1)
+    right_edge[:, 1]  = isoparam.isoparam_y(y_nodes, 1., eta)
+    top_edge[:, 1]    = isoparam.isoparam_y(y_nodes, xi, 1.)
+    
+    # Plot edges
+    utils.plot_line(left_edge, axes_handler, grid_width, grid_color)
+    utils.plot_line(bottom_edge, axes_handler, grid_width, grid_color)
+    utils.plot_line(right_edge, axes_handler, grid_width, grid_color)
+    utils.plot_line(top_edge, axes_handler, grid_width, grid_color)
+    
+    return
+
+def plot_mesh_grid(nodes, elements, xi_LGL, eta_LGL, axes_handler):
+    '''
+    Plots the mesh grid.
+    
+    Parameters
+    ----------
+    
+    nodes : np.ndarray [N, 2]
+            Array of nodes in the mesh. First column and the second column are
+            the :math:`x` and :math:`y` coordinates respectivily.
+            
+    elements : np.ndarray [N_e, 8]
+               Array of elements.
+               
+    xi_LGL  : np.array [N_LGL]
+              LGL points on the :math:`\\xi` axis
+
+    eta_LGL  : np.array [N_LGL]
```

Comment:
Alignment of the colons

Reply:
checked at all other places and modified the documentations accordingly.